src/orange/detail/UnitInserter.cc

```diff
@@ -285,8 +285,11 @@ UniverseId UnitInserter::operator()(UnitInput const& inp)
     {
         unit.background = LocalVolumeId(inp.volumes.size() - 1);
     }
+
+    // Simple safety if all volumes provide support, excluding the external
+    // volume, which appears first in the list
     unit.simple_safety = std::all_of(
-        vol_records.begin(), vol_records.end(), [](VolumeRecord const& v) {
+        vol_records.begin() + 1, vol_records.end(), [](VolumeRecord const& v) {
```

Comment:
This is above-board if we guarantee that external is first, but I am not sure if that's a good idea.

Reply:
See `OrangeTrackView::is_outside`. Maybe we should define: in OrangeData:
```c++
static inline constexpr LocalVolumeId local_orange_outside_volume{0};
```
and in your code here add:
```c++
static_assert(local_orange_outside_volume == LocalVolumeId{0});
```
and also use it in orange track data:
```c++
CELER_FUNCTION bool OrangeTrackView::is_outside() const
{
    // Zeroth volume in outermost universe is always the exterior by
    // construction in ORANGE
    auto lsa = this->make_lsa(LevelId{0});
    return lsa.vol() == local_orange_outside_volume;
}
```